arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "InsightsManagementClient",
-    "version": "2015-04-01"
+    "version": "2016-03-01"
```

Comment:
Yes, the file still contains different version for the api: 2016-03-01, 2015-04-01, 2015-07-01. I will correct the folder names when I split the files. For the moment the folder name refers only to the oldest api-version contained in the files.